CMSgov/cmcs-eregulations
# EREGCSC-1599 -- Last ECFR Parse Date in Footer from API

Resolves [EREGCSC-1599](https://jiraent.cms.gov/browse/EREGCSC-15990

**Description**
In order to test API calls inside of the application, we are going to call the `ecfr_parser_result` API endpoint and display the `end` property of the response object as Last Parser Success Date in the footer.

**This pull request changes...**

- Creates new Vue component to make async call to `ecfr_parser_result` API endpoint and display the result as a date.
- Updates Rollup configuration and Django templates to use this new Vue component.
- Adds utility methods to make API call and format response

**Steps to manually verify this change**

1. Go to experimental deployment and scroll down to the footer
2. Ensure "Regulations up to date from eCFR as of" has a date at the end of the line
3. Inspect the network tab to see that a call to `ecfr_parser_result` was made
